scripts/superdesk/editor2/editor.js

```diff
@@ -1334,26 +1334,47 @@ angular.module('superdesk.editor2', [
         };
     }])
     .run(['embedService', 'iframelyService', function(embedService, iframelyService) {
-        var pattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
-        var loader = '//snappa.embed.pressassociation.io/playbuzz.js';
-        var html = [
+        var playBuzzPattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
+        var playBuzzlLoader = '//snappa.embed.pressassociation.io/playbuzz.js';
```

Comment:
The main advantage to let PA to host it, is that they will update the script if needed (there are the author of this script and they use it often)
But if you think it's important to fork it in SD, I could do it. Just confirm it :)